Makefile

```diff
@@ -81,19 +78,19 @@ hugo-dependencies:
 	@if [[ ! -f $(HUGO) ]]; then \
 		$(MKDIR) $(HUGO_PATH); \
 		cd $(HUGO_PATH); \
-		ext="zip"; \
-		if [ "$(ARCH)" == "linux" ]; then ext="tar.gz"; fi; \
+		ext="tar.gz"; \
+		if [ "$(ARCH)" == "windows" ]; then ext="zip"; fi; \
 		file="hugo.$${ext}"; \
 		$(CURL) https://$(HUGO_URL)/releases/download/v$(HUGO_VERSION)/$(HUGO_URL_NAME).$${ext} -o $${file}; \
-		if [ "$(ARCH)" == "linux" ]; then tar -xvzf $${file}; else unzip $${file}; fi; \
+		if [ "$(ARCH)" == "windows" ]; then unzip $${file}; else tar -xvzf $${file}; fi; \
```

Comment:
This was initially part of the autohugo project, which was a generic thing. Now that it's for us idk how much value adds to support a platform we are never ever using.